File: Introduction to VB/Form2.vb
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Namespace Introduction_to_VB
    Public Partial Class Form2
        Inherits Form
        Public Shared Instance As Form2
        Public Sub New()

            InitializeComponent()
            Instance = Me
        End Sub

        Private Sub Form2_Load(sender As Object, e As EventArgs)

        End Sub




        Private Sub radioButton2_CheckedChanged(sender As Object, e As EventArgs)
            If blue.Checked Then

                textBox1.ForeColor = Color.Blue
            End If
        End Sub

        Private Sub radioButton3_CheckedChanged(sender As Object, e As EventArgs)
            If red.Checked Then

                textBox1.ForeColor = Color.Red
            End If
        End Sub

        Private Sub radioButton3_CheckedChanged_1(sender As Object, e As EventArgs)
            If tahoma.Checked Then

                textBox1.Font = New Font("Tahoma", textBox1.Font.Size)
            End If
        End Sub

        Private Sub checkBox3_CheckedChanged(sender As Object, e As EventArgs)
            If bolditalic.Checked Then

                textBox1.Font = New Font(textBox1.Font, Drawing.FontStyle.Bold Or Drawing.FontStyle.Italic)

            ElseIf bold.Checked Then

                textBox1.Font = New Font(textBox1.Font, Drawing.FontStyle.Bold)

            ElseIf italic.Checked Then

                textBox1.Font = New Font(textBox1.Font, Drawing.FontStyle.Italic)
            Else

                textBox1.Font = New Font(textBox1.Font, Drawing.FontStyle.Regular)
            End If
        End Sub

        Private Sub label1_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub label1_Click_1(sender As Object, e As EventArgs)

        End Sub

        Private Sub loadpicture_Click(sender As Object, e As EventArgs)
            Dim openFileDialog As OpenFileDialog = New OpenFileDialog()
            openFileDialog.Filter = "Image files (*.jpg, *.jpeg, *.png, *.gif) | *.jpg; *.jpeg; *.png; *.gif"

            If openFileDialog.ShowDialog() = DialogResult.OK Then

                pictureBox1.ImageLocation = openFileDialog.FileName
            End If
        End Sub

        Private Sub pictureBox1_Click(sender As Object, e As EventArgs)

        End Sub

        Private Sub textBox1_TextChanged(sender As Object, e As EventArgs)

        End Sub

        Private Sub garamond_CheckedChanged(sender As Object, e As EventArgs)
            If garamond.Checked Then

                textBox1.Font = New Font("Garamond", textBox1.Font.Size)
            End If
        End Sub

        Private Sub magneto_CheckedChanged(sender As Object, e As EventArgs)
            If magneto.Checked Then

                textBox1.Font = New Font("Magneto", textBox1.Font.Size)
            End If
        End Sub

        Private Sub italic_CheckedChanged(sender As Object, e As EventArgs)
            If italic.Checked Then

                textBox1.Font = New Font(textBox1.Font, Drawing.FontStyle.Italic)
            Else

                textBox1.Font = New Font(textBox1.Font, textBox1.Font.Style And Not Drawing.FontStyle.Italic)
            End If
        End Sub

        Private Sub bold_CheckedChanged(sender As Object, e As EventArgs)
            If bold.Checked Then

                textBox1.Font = New Font(textBox1.Font, Drawing.FontStyle.Bold)
            Else

                textBox1.Font = New Font(textBox1.Font, textBox1.Font.Style And Not Drawing.FontStyle.Bold)
            End If
        End Sub

        Private Sub exit_Click(sender As Object, e As EventArgs)
            Dim form1 As Form1 = New Form1()
            form1.Show()
        End Sub

        Private Sub green_CheckedChanged(sender As Object, e As EventArgs)
            If green.Checked Then

                textBox1.ForeColor = Color.Green
            End If
        End Sub
    End Class
End Namespace

File: Introduction to VB/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Introduction_to_VB
{
    public partial class Form2 : Form
    {
        public static Form2 Instance;
        public Form2()
        {

            InitializeComponent();
            Instance = this;
        }

        private void Form2_Load(object sender, EventArgs e)
        {

        }

      
        

        private void radioButton2_CheckedChanged(object sender, EventArgs e)
        {
            if (blue.Checked)
            {
               
                textBox1.ForeColor = Color.Blue;
            }
        }

        private void radioButton3_CheckedChanged(object sender, EventArgs e)
        {
            if (red.Checked)
            {
               
                textBox1.ForeColor = Color.Red;
            }
        }

        private void radioButton3_CheckedChanged_1(object sender, EventArgs e)
        {
            if (tahoma.Checked)
            {
                
                textBox1.Font = new Font("Tahoma", textBox1.Font.Size);
            }
        }

        private void checkBox3_CheckedChanged(object sender, EventArgs e)
        {
            if (bolditalic.Checked)
            {
               
                textBox1.Font = new Font(textBox1.Font, FontStyle.Bold | FontStyle.Italic);
            }
            
            else if (bold.Checked)
            {
                
                textBox1.Font = new Font(textBox1.Font, FontStyle.Bold);
            }
                   
            else if (italic.Checked)
            {
                       
                textBox1.Font = new Font(textBox1.Font, FontStyle.Italic);
            }
            else
            {
                  
                textBox1.Font = new Font(textBox1.Font, FontStyle.Regular);
            }
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }

        private void label1_Click_1(object sender, EventArgs e)
        {

        }

        private void loadpicture_Click(object sender, EventArgs e)
        {
            OpenFileDialog openFileDialog = new OpenFileDialog();
            openFileDialog.Filter = "Image files (*.jpg, *.jpeg, *.png, *.gif) | *.jpg; *.jpeg; *.png; *.gif";

            if (openFileDialog.ShowDialog() == DialogResult.OK)
            {
               
                pictureBox1.ImageLocation = openFileDialog.FileName;
            }
        }

        private void pictureBox1_Click(object sender, EventArgs e)
        {

        }

        private void textBox1_TextChanged(object sender, EventArgs e)
        {

        }

        private void garamond_CheckedChanged(object sender, EventArgs e)
        {
            if (garamond.Checked) 
            {
               
                textBox1.Font = new Font("Garamond", textBox1.Font.Size);
            }
        }

        private void magneto_CheckedChanged(object sender, EventArgs e)
        {
            if (magneto.Checked) 
            {
               
                textBox1.Font = new Font("Magneto", textBox1.Font.Size);
            }
        }

        private void italic_CheckedChanged(object sender, EventArgs e)
        {
            if (italic.Checked) 
            {
               
                textBox1.Font = new Font(textBox1.Font, FontStyle.Italic);
            }
            else
            {
                
                textBox1.Font = new Font(textBox1.Font, textBox1.Font.Style & ~FontStyle.Italic);
            }
        }

        private void bold_CheckedChanged(object sender, EventArgs e)
        {
            if (bold.Checked) 
            {
                
                textBox1.Font = new Font(textBox1.Font, FontStyle.Bold);
            }
            else
            {
                
                textBox1.Font = new Font(textBox1.Font, textBox1.Font.Style & ~FontStyle.Bold);
            }
        }

        private void exit_Click(object sender, EventArgs e)
        {
            Form1 form1 = new Form1();
            form1.Show();
        }

        private void green_CheckedChanged(object sender, EventArgs e)
        {
            if (green.Checked)
            {
                
                textBox1.ForeColor = Color.Green;
            }
        }
    }
}
